.maestro/flows/local.yaml

```
@@ -3,58 +3,20 @@ tags: [critical]
-    env: { asset: USDC, to: "${output.account}", amount: "420" }
-- tapOn: Home
-- runFlow: ../subflows/readPortfolio.yaml
+- assertVisible: ${output.account.slice(0, 6)}…${output.account.slice(-4)}
 - runFlow:
-    when: { true: "${output.portfolio === output.portfolioBefore}" }
-    commands: [{ tapOn: Home }, { runFlow: ../subflows/readPortfolio.yaml }]
-- assertTrue: ${output.portfolio - output.portfolioBefore === 420}
-- evalScript: ${output.portfolioBefore = output.portfolio}
-- runScript: ../dist/getAccount.js
+    file: ../subflows/mint.yaml
+    env: { asset: USDC, to: "${output.account}", amount: "420" }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)